src/platz1de/EasyEdit/utils/ItemInfoUtil.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace platz1de\EasyEdit\utils;
+
+use platz1de\EasyEdit\session\Session;
+use platz1de\EasyEdit\convert\ItemConvertor;
+use pocketmine\item\Item;
+use pocketmine\nbt\tag\ByteArrayTag;
+use pocketmine\nbt\tag\CompoundTag;
+use pocketmine\nbt\tag\ImmutableTag;
+use pocketmine\nbt\tag\IntArrayTag;
+use pocketmine\nbt\tag\ListTag;
+use pocketmine\nbt\tag\StringTag;
+use pocketmine\world\format\io\GlobalItemDataHandlers;
+use platz1de\EasyEdit\thread\EditThread;
+
+class ItemInfoUtil
+{
+
+    public static function convertNbtToPrettyString(CompoundTag | ListTag $nbt): string
+    {
+        $isCompoundTag = $nbt instanceof CompoundTag;
+        $openingChar = $isCompoundTag ? "{" : "[";
+        $stringified = $openingChar;
+
+        $idx = 0;
+		foreach($nbt->getValue() as $name => $tag) {
+            $value = null;
+            if ($tag instanceof CompoundTag || $tag instanceof ListTag) {
+                $value = self::convertNbtToPrettyString($tag);
+            } else {
+                $tagAsString = $tag->toString();
+                $value = substr($tagAsString, strpos($tagAsString,"=") + 1);
+            }
+
+            $valueColor = "§r";
```

Comment:
Use the [TextFormat](https://github.com/pmmp/PocketMine-MP/blob/stable/src/utils/TextFormat.php) constants instead of magic codes